### ISSUE TITLE
[Keyboard Navigation - Bot Webchat - Home]: Keyboard focus indicator is not visible on 'upload file' and 'send' buttons.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4014]
User Experience:
If focus indicator is not visible on 'upload file' and 'Send' buttons, then users with motor disabilities who depend on keyboard for navigation will not be able to identify the focus on that control which can prevent them from performing actions which are important.

Repro steps:
1. Open URL: Web Chat: Full-featured bundle (microsoft.github.io)
2. Web chat bundle will open.
3. Navigate on the web chat bot.
4. Check and verify that step 3 is accessible and MAS compliant.
5. Verify the issue.

Actual Result:
Keyboard focus indicator is not visible on 'upload file' and 'send' buttons.

Expected Result:
Keyboard focus indicator should be clearly visible on 'upload file' and 'send' buttons.

MAS Reference:
WCAG2.4.7 – Focus Visible

Test Environment:
OS: Win 10 21H1
Build: OS Build 19043.1147
Browser: Edge Chromium Version 94.0.972.2 (Official build) dev (64-bit)
#URL: Web Chat: Full-featured bundle (microsoft.github.io)

**Comment by KattaBhargav** [comment]
A11yMAS;A11ySev2;HCL;MAS2.4.7;Benchmark;Accessibility;Win10-Edge(Chromium);E2E-HCL-BotFrameworkWebChat-Aug21

**Comment by compulim** [comment]
I think we need to tweak the color or form of the focus indicator. We do have the focus indicator.

![image](https://user-images.githubusercontent.com/1622400/129113108-188811df-5f26-4d5e-894c-87c960955dad.png)

![image](https://user-images.githubusercontent.com/1622400/129113175-2273e876-ccdb-405b-9a2c-5151d04a29ae.png)

**Comment by compulim** [comment]
We have fixed this in PR #4035. Here is the quick demo after the fix.

![recording (28)](https://user-images.githubusercontent.com/1622400/134992308-f47d7999-ba36-46cf-a39f-79f7df2a5730.gif)

**Comment by v-shikch** [comment]
As verified issue on URL: Web Chat: Full-featured bundle (microsoft.github.io) on Browser: Edge Chromium Version 96.0.1028.0 (Official build) dev (64-bit), here issue is still repro at our end. As keyboard focus is still not appearing round the highlighted 'send' and 'upload file' button controls.

**Comment by compulim** [comment]
@v-shikch Since the fix is not out on production yet, could you try it out on this page?

https://corinagum.github.io/WebChat-release-testing/03.a.renderwebchat-using-es5-bundle/index.html

This page use the daily development build version of Web Chat.

**Comment by v-shikch** [comment]
Verified issue on below env:

OS: Win 10 21H1
Build: OS Build 19043.1147
Browser: Edge Chromium Version 96.0.1028.0 (Official build) dev (64-bit)
#URL: https://corinagum.github.io/WebChat-release-testing/03.a.renderwebchat-using-es5-bundle/index.html

here issue is fixed, now focus indicator is visible on the upload file and send button. Hence closing the issue


;#Closed;#Regression
